src/FtpClient.php

```diff
@@ -485,20 +485,21 @@ public function removeDirectory(string $path) : void
     /**
      * Returns the size of the given file.
      *
-     * @param string $path The remote file path.
+     * @param string $remoteFile The remote file.
      *
      * @return int The file size.
      *
      * @throws FtpException If not connected, or an error occurs.
      */
-    public function getSize(string $path) : int
+    public function getFileSize(string $remoteFile) : int
```

Comment:
This is a breaking change, why change the method name?

Reply:
As I said in the PR description, this method doesn't work with directories anyway, the name `getSize()` may be confused to lib users, a user may think the method doesn't work at all if he uses this with a remote directory, so changing the name to `getFileSize()` is more semantic. So what you think?

Reply:
This makes sense, but will require a version bump. What's the current behaviour if called on a dir?

Reply:
I answer on that already, please check our discussion below.

Reply:
And yes I think is required a bump version may be `v0.3.0`.